metadata.rb

```diff
@@ -1,8 +1,15 @@
-name             'timesync'
-maintainer       'Oregon State University'
+name 'timesync'
```

Comment:
I think that rubocop config should be in the code generator cookbook. Mind if I make a PR adding it? I assume that even though that repo is private, the rubocop config doesn't really need to be.